atc/policy/checker.go

```diff
@@ -0,0 +1,160 @@
+package policy
+
+import (
+	"fmt"
+	"strings"
+
+	"code.cloudfoundry.org/lager"
+	"github.com/jessevdk/go-flags"
+)
+
+const ActionUsingImage = "UsingImage"
+
+type PolicyCheckNotPass struct{}
+
+func (e PolicyCheckNotPass) Error() string {
+	return "policy check not pass"
```

Comment:
This sounds a little better:

```suggestion
	return "policy check rejected"
```

(Plus renaming the type to match if that message is OK.)

Reply:
Fixed.